gen/roundtrip_test.go

```diff
@@ -81,3 +83,53 @@ func assertBinaryRoundTrip(t *testing.T, w wire.Value, message string) (wire.Val
 
 	return newV, true
 }
+
+func testRoundTripCombos(t *testing.T, x thriftType, v wire.Value, msg string) {
+	t.Helper()
+
+	useStreaming := []struct {
+		encode bool
+		decode bool
+	}{
+		{false, false},
+		//{false, true},
```

Comment:
Are these cases supposed to be commented out vs removed?

Reply:
Commented out, this will eventually be merged with #496 which will provide the other half of the test suite.

Reply:
👍 yep - I'll be taking care of the merging.